test/release/examples/primers/parIters.chpl

```diff
@@ -472,7 +472,7 @@ Closing notes
 // routine works only for unstrided ranges with the default index type
 // of ``int``.  These constraints could be relaxed with more effort.
 // 
-proc computeChunk(r: range, myChunk, numChunks) where r.stridable == false {
+proc computeChunk(r: range, myChunk, numChunks) {
```

Comment:
does this now work with unstrided ranges?

Reply:
It does not. I removed the where clause because the formal `r: range` already requires an unstrided range.